crates/astria-sequencer-validation/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-ct-merkle = "0.1"
+ct-merkle = { version = "0.1", features = ["serde"] }
```

Comment:
`ct-merkle` is imported here, but there is a `ct-merkle` in the workspace cargo toml that refers to a git repo under the astria org.